app/_hub/reedelk/reedelk-transformer/index.md

```diff
@@ -0,0 +1,205 @@
+---
+name: Reedelk Transformer
+
+publisher: Reedelk
+
+categories:
+  - transformations
+
+type:
+  plugin        
+
+desc: Kong plugin to transform Reedelk requests and responses
+
+description: |
+  The Reedelk  transformer plugin allows transforming the upstream request body or
+  downstream response body by invoking a Reedelk REST flow before hitting the
+  upstream server, or before sending the downstream response back to the client.
+  The plugin allows applying upstream and downstream transformations together as well.
```

Comment:
```suggestion
  The Reedelk Transformer plugin transforms the upstream request body or
  downstream response body by invoking a Reedelk REST flow before hitting the
  upstream server, or before sending the downstream response back to the client.
  The plugin can apply both upstream and downstream transformations in the same flow.
```
Initial caps for the plugin name, since it's a proper name. 
Last sentence: not entirely sure that the phrasing is correct - maybe "to the same data set", or something along those lines? 

I'd avoid using the phrasing of "allows doing something" - there's usually a more succinct way of saying the same thing.

Reply:
This is much improved, thanks!